framework/src/modules/interoperability/base_interoperability_store.ts

```diff
@@ -40,8 +40,8 @@ export abstract class BaseInteroperabilityStore {
 	// To be implemented in base class
 	public abstract apply(ccu: CCUpdateParams, ccm: CCMsg): Promise<void>;
 	public abstract appendToInboxTree(chainID: number, appendData: Buffer): Promise<void>;
-	public abstract appendToOutboxTree(chainID: number, appendData: Buffer): Promise<void>;
-	public abstract addToOutbox(chainID: Buffer, ccm: CCMsg): Promise<void>;
+	public abstract appendToOutboxTree(chainID: Buffer, appendData: Buffer): Promise<boolean>;
+	public abstract addToOutbox(chainID: Buffer, ccm: CCMsg): Promise<boolean>;
```

Comment:
It was not mentioned in the issue but if you can move the implementations of `appendToOutboxTree` and `addToOutbox` to `BaseInteroperabilityStore` class as the code will be the same for mainchain and sidechain.

Reply:
Similar to https://github.com/LiskHQ/lisk-sdk/blob/72a8ede36f90bb334b386d91828a5c4766b5434e/framework/src/modules/interoperability/base_interoperability_store.ts#L48-L70